modules/admin_manual/pages/configuration/user/user_roles.adoc

```diff
@@ -66,7 +66,7 @@ backend).
 == Federated User
 
 * Is not an internal user.
-* Can trust xref:faq/index.adoc#what-is-a-federated-system[a federated system].
+* Can trust xref:troubleshooting/faq.adoc#what-is-a-federated-system[a federated system].
```

Comment:
Because that is moved, we have to change:
`troubleshooting/faq.adoc` --> `faq/index.adoc`